src/igvfd/schemas/construct_library.json

```diff
@@ -0,0 +1,317 @@
+                ]
+            },
+            "comment": "Specification of crispr_library_details is mutually exclusive with specification of mpra_library_details."
+        },
+        "mpra_library_details": {
+            "not": {
+                "required": [
+                    "crispr_library_details"
+                ]
+            },
+            "comment": "Specification of mpra_library_details is mutually exclusive with specification of crispr_library_details."
+        },
+        "scope": {
+            "oneOf": [
+                {
+                    "required": [
+                        "targeted_genes"
+                    ],
+                    "properties": {
+                        "scope": {
+                            "enum": [
+                                "genes"
+                            ]
+                        }
+                    }
+                },
+                {
+                    "required": [
+                        "targeted_regions"
+                    ],
+                    "properties": {
+                        "scope": {
+                            "enum": [
+                                "chromosomal regions"
+                            ]
+                        }
+                    }
+                },
+                {
+                    "not": {
+                        "properties": {
+                            "scope": {
+                                "enum": [
+                                    "genes",
+                                    "chromosomal regions"
+                                ]
+                            }
+                        }
+                    }
+                }
+            ]
+        }
+    },
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "accession": {
+            "accessionType": "FS"
+        },
+        "scope": {
+            "title": "Library Construct Scope",
+            "description": "The scope or scale that this construct library is designed to target. If the scope is across gene(s) or chromosomal region(s), these will need to be specified in the targeted_genes or targeted_regions property.",
+            "type": "string",
+            "enum": [
+                "genes",
+                "chromosomal regions",
+                "genome-wide",
+                "population"
+            ]
+        },
+        "targeted_genes": {
+            "title": "Targeted Genes",
+            "description": "The specific gene(s) this construct library was designed to target.",
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "comment": "See gene.json for a list of available identifiers.",
+                "type": "string",
+                "linkTo": "Gene"
+            }
+        },
+        "targeted_regions": {
+            "title": "Targeted Loci",
```

Comment:
I think we want singular, not plural for property names

Reply:
Understood, all are singularized